Automattic/wp-calypso
# Limit Global Styles: Replace JS images with dynamic webpack image paths

#### Proposed Changes

Ensures that the image displayed in the GS modal has the correct URL by dynamically setting the public path of webpack. This follows the same approach took by [`whats-new`](https://github.com/Automattic/wp-calypso/blob/0b3d47279751a65f059998bba607c6ef90a2c75b/apps/editing-toolkit/editing-toolkit-plugin/whats-new/src/public-path.js) and [`wpcom-block-editor-nux`](https://github.com/Automattic/wp-calypso/blob/0b3d47279751a65f059998bba607c6ef90a2c75b/apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nux/src/public-path.js).

Before | After
--- | ---
<img width="995" alt="Screen Shot 2022-09-16 at 12 32 39" src="https://user-images.githubusercontent.com/1233880/190632884-43b70bd6-1608-4e0b-b142-70b9efdc0df6.png"> | <img width="969" alt="Screen Shot 2022-09-16 at 14 59 40" src="https://user-images.githubusercontent.com/1233880/190644588-a3992714-b39a-464d-a934-69226e02a4db.png">

It also updates the image as per @SaxonF request in p1663545533696099/1663140264.299019-slack-CJS75TX3R

#### Testing Instructions

The issue is only reproducible on production (i.e. on a non-sandboxed site) so this is hard to test, but at least you can apply the changes to your sandbox and verify that the modal keeps working.

- Apply these changes to your sandbox: `install-plugin.sh editing-toolkit fix/limit-global-styles-assets-url`
- Go to https://wordpress.com/start and create a new free site
- Add the `wpcom-limit-global-styles` blog sticker to the new site from the Blog RC
- Sandbox the new site
- Go to https://wordpress.com
- Go to Appearance > Themes
- Activate a block theme such as Twenty-twenty Two
- Go to Appearance > Editor
- Open the Global Styles sidebar
- Make sure the paid Global Styles modal shows up and the image is visible

#### Pre-merge Checklist
